engine/src/main/java/org/hibernate/validator/resourceloading/PlatformResourceBundleLocator.java

```diff
@@ -77,12 +90,66 @@ public PlatformResourceBundleLocator(String bundleName, ClassLoader classLoader)
 	 * @since 5.2
 	 */
 	public PlatformResourceBundleLocator(String bundleName, ClassLoader classLoader, boolean aggregate) {
+		this( bundleName, Collections.emptySet(), classLoader, aggregate );
+	}
+
+	/**
+	 * Creates a new {@link PlatformResourceBundleLocator}.
+	 *
+	 * @param bundleName the name of the bundle to load
+	 * @param localesToInitialize The set of locales to initialize at bootstrap.
```

Comment:
The -> the I see that we usually start param description with the lowercase (there's a few more below)

Reply:
Fixed.